README.md

```diff
@@ -9,23 +9,32 @@ Groupie lets you treat your content as logical groups and handles change notific
 # Try it out:
 
 ```gradle
-implementation "com.xwray:groupie:2.7.0"
+implementation "com.xwray:groupie:$groupie_version"
 ```
 
 Groupie includes a module for Kotlin and Kotlin Android extensions. Never write a ViewHolder again—Kotlin generates view references and Groupie uses a generic holder. [Setup here.](#kotlin) 
 
 ```gradle
-implementation "com.xwray:groupie:2.7.0"
-implementation "com.xwray:groupie-kotlin-android-extensions:2.7.0"
+implementation "com.xwray:groupie:$groupie_version"
+implementation "com.xwray:groupie-kotlin-android-extensions:$groupie_version"
 ```
 
 Groupie also supports Android's [data binding](https://developer.android.com/topic/libraries/data-binding/index.html) to generate view holders. [Setup here.](#data-binding)
 
 ```gradle
-implementation "com.xwray:groupie:2.7.0"
-implementation "com.xwray:groupie-databinding:2.7.0" 
+implementation "com.xwray:groupie:$groupie_version"
+implementation "com.xwray:groupie-databinding:$groupie_version" 
 ```
 
+Groupie also has a support module for Android's [view binding](https://developer.android.com/topic/libraries/view-binding). This module also supports Android data binding, so if your project uses both data vinding and view binding, you don't have to add the dependency on the data binding support module. [Setup here.](#view-binding)
+
+### Note:
+`groupie-viewbinding` can also be used for the project using only data binding, but in this case the version of Android Gradle Plugin must be 3.6.0 or more.
```

Comment:
correct to: 
`groupie-viewbinding` can also be used for projects using only data binding when using Android Gradle Plugin  3.6.0 or higher